components/binding-usage-controller/docs/assets/architecture.html

```diff
@@ -5,7 +5,7 @@
e=\&quot;Page-1\&quot;&gt;7Vxbd+I2EP41PMKxfIXHBLLdnqY9OU33tH0UtgBtjE1tk8D++kq2ZFsXg8GGZJPwsEvGsjQajT59Mx4zsKbr3S8J3Kx+jwMUDkwj2A2s2cA0gW2NyX9UsmcS4IBCskxwwGSV4BH/QExoMOkWBygVGmZxHGZ4Iwr9OIqQnwkymCTxi9hsEYfiqBu4ZCMaleDRhyFSmv2Ng2xVSMdOrfVXhJcrPjIw2JU59J+WSbyN2HgD01rkn+LyGvK+WPt0BYP4pSay7gbWNInjrPi23k1RSI3LzVbc96Xhaql3gqKszQ2WXdzxDMMt4irnimV7bgw6n43aMRvrGSUZ2ulWBc5DJM+MuAyK1yhL9qTdjuvA3IV5izU2R04heamMb1vMZKua4c0xE0K24Muy92rS5Aubt94Gjne2DUrznW8EvkWMljP2epixZWpm7IZkiNsAPwszd//bUl+8JdPLhjDEy2hg3dDpERugpLpOvi3Z/3k/6QZGsmwuCxYxMaRuNHphmOawQAcDzmanDjVDmzDer+lisP7IvIsuxWGIWBmayHQqatVOWt5MxLn5RGlbi4ZokZ1oz0/FWioWwjkK0/yug/poFb+EhnC9IV+iebppNqXQZpuiYSkrWqPkGfvoFkcBjpbfUnImfft1VrYpesm1OHofaZOgNN4mPirALMVxJPX0KgspmcmAQYAzohsM7+mKgnJ+vOOE4Fm9o1dUzpSVSwlQDwnkXk3BK7o0jKI4g3T20i672LyKk1UWzmPCOWWhYoEs0CoQ4ggN+WFLNbDJ6jnNKmwSRQF2oi2gL3b9FYXPKMM+VHtj+3Fe7Mct3Y/piAl9mMEwXo6e9ms48nf0gNcfdVkCfXL3EEeLOFnnC1G6H2+vUVcUVYodhos/4JodzDj6TjgOCu4LfJURQ3VzjQ5EStdDFeocV7vAVK7zBtn9JE6X03REKZFBLr+scIYeN8XCvZC4hshW2ZpQqBmgZuBE3eDHyW1J9adxGCfkUhRHiDbNkvgJcSGJAGbmzJlNyis8oqDdLnAY1lp+yT+lsnWqWbJUmWo282qBVTumSjBNTyWYYDzugVK/flRhm2JQYavzBzqC7bndp+820uv62cm2FGk4JTs5icMwJ9T6Y7bRd8Flfde0pzfT6VHfZbdr3La113I/HRmu8Gm5bMZoYtU+Zg9R0hvwYst2BTcGrqWJE23NNnYmPVjAOD8yblr49iYQpzgy6h/3QjM2dbmAy0fG3QLhkupxwUMcPBBCj+qBsS4CbgiWPwPjvhVLkZ+g7E+0OB6CXilYiUoOp3hPggKcDuf8dDroQZcLW1qt6VVUInTaX91rMggXX6c3FPR/hIXuWaWfOtwoj/7WxM3UZbB1zMTqg5noKPaVudnQdA2BnJnuRENWHdUEZba/iwnst0BPxSDL4Y/56gYYA40B7D6eYow7c7XDOKABmPn5LG0ablNCDB8LJJ6GME0PoclJebvrUKJrjsLoxynHbiPXIWeo8YT2xeIoKeQOHWtEzBnpSCwZ3Yva4kUl03yxOdRy1gdGeFcnn31yysLUwL6ryVEA1+kB9ls8rUbBEj2yP+MkW8VL6it3lfRWXJKa+dEOZ/8wMf3+L/0+clqsBUyyG1p6Icm+YDqVor8o4C18Cn/YL4SsCR3+O8qyPasHgdssJqJqAvdxvOFeIi/9xJlMb2zt0jeef4z81lN2ROMl4ocUk1FrHnQDzgeMkWFIWdchYH8nKIQZfhbrS3R+wLp/iHGeEuBN4sUij+crRyGmhPtaiw29I9V0wdWzbUE1a2zVuzu1PflSaFB5bWmSdjnidlnSFxp9NZ4mkuPT8130ZxLi4B+MzlAfZFYirZ3bgTOjYFrQgRnL21i3dONjH4Y37MIaB0G+aRp2QFXjUzCBmrxIVulhpimffzoPNUbA49Ys/a6b27GuDV6IU3YLRhKGqY55hi/wCqCeMSzHLY5i4KNg2FG86ugUwPBGQqpddD1nLPVYYCrrpJObOGo+eoFq8FAjIRVFak2OQv4Qsw2neXWk0SAKj4f6QBTTm9iXOrnOWfnmWI/lzv3SRlXgVdZeVqIW0Zuti97qcYKYNpdxKwzxJkXHiS5MN0Xp6gLvKIqpeGDcuHc1pOB4YKp0tmzZiBPtHxsPbTGrYY3VYkzOcoSY3upObvkjnIPkVofA9QCiEVarc2Ey8QR+O3b49QeUYKI23Z4z/cHdAXhrFnQ08QGXdcVnTzy0bUN6ol+orgCy0pHliJ5gy7WnDch+KjMtq3glhZv0Utp7QpFzZ2bq6h57HgcbRvYU6MjBnJojCvJ/80op1BZPzjpoCOsoAK1AGna85Nku9dTJKCOhnJtWEP2V05OhrXLZchv1gDFgLGOMp2AMAJotYvaQN3SBZnl7A5mmCKAZfMDYe0fgM5FW1vXOBB+p8kFZ+guBj+UaB/WS25MZ9ws+5iszHfPDMB1gN5xvV2A6ru7pzTWX2T6Ys3/fC2+OxdSWObniwqtpLz9BOSF4qyf/gSzW6aaX9pw5AZqXvXQHVC9Hvy6+uOaucz7OHgOeCK6aN9wutccOFOBeruis5xq+R13Fzmc136diB8sM6z7zWW74c1eh6TTofbRtioLb/ZXrGknDJ0xfkdc6RKB7r/dapj/Zcf0Qn6ipdNL/XNURp6eTLSAF2EB9haFMWYpvInndqYCny+VJK1BUyckWa3ylpb8CQkf84QNNFKKtniuLSzoZRpcFu3D1XHEA9UaRclL0GwWSw7uvU5nbGXV4BzCjM7b1ViTY9tZ3hV4lGrSu7QIT4Vl3+WLZ0VqvPkp8PU1i+oSyCDBoURZBl6JtxUIjFh6ppOJY019lQmsLqr9vQxhPP6mWyz7N52vfy9N8w5mIzwRZ5+cm+/nmMC9SDeRZr5ylsWrg976zNAAA0S8ulwklf1a/a1V4RfXrYdbd/w==&lt;/diagram&gt;&lt;/mxfile&gt;&quot;}"></div>
```

Comment:
Adjust the diagram as discussed.  Add **labelsPath** for `UK` which will match the Deployment label path

Reply:
done